README.md

```diff
@@ -16,6 +16,126 @@ Stride is a blockchain ("zone") that provides liquidity for staked assets. Using
 
 ![](https://drive.google.com/uc?id=1RuK2YeMH7O6P9-8ro_ybOg5n79ySwFjN)
 
+## Setting up a node
+Make sure your system is updated and set the system parameters correctly:
+```
+sudo apt-get update
+sudo apt-get upgrade -y
+sudo apt-get install -y build-essential curl wget jq
+sudo su -c "echo 'fs.file-max = 65536' >> /etc/sysctl.conf"
+sudo sysctl -p
+```
+
+### Install go
+First remove any existing old Go installation:
+```
+sudo rm -rf /usr/local/go
```

Comment:
This command feels slightly dangerous to run? Not sure

Reply:
It only deletes an old folder of go (if it exists). Not dangerous, have done it a couple of times already :P
And the command following immediately is installing a newer version of go.

Reply:
Agreed on it not being dangerous - it's stated in the go installation [docs](https://go.dev/doc/install)